rockets/CMakeLists.txt

```diff
@@ -20,9 +20,7 @@ list(APPEND CPPCHECK_EXTRA_ARGS
   --suppress=*:${CMAKE_CURRENT_SOURCE_DIR}/json.hpp)
 
 # bug while checking json.hpp, --suppress does not remove the error exit code
```

Comment:
remove those two lines at all?

Reply:
I don't know, if the parent project does not (want to) disable cppcheck the tests fail because of this false positive...

Reply:
Ah true, it's a global flag. Ok then.